SharpDenizenTools/MetaHandlers/MetaDocs.cs

```diff
@@ -115,6 +99,23 @@ public class MetaDocs
         /// <summary>Set of raw adjustable keys.</summary>
         public HashSet<string> RawAdjustables = [];
 
+        /// <inheritdoc/>
+        public MetaDocs()
+        {
+            // Extensions explicitly first
+            META_TYPE_EXTENSION = new(this, "Extension", null, 256, () => new MetaExtension());
+            META_TYPE_COMMAND = new(this, "Command", "Commands", 512, () => new MetaCommand());
+            META_TYPE_MECHANISM = new(this, "Mechanism", "Mechanisms", 1024, () => new MetaMechanism());
+            META_TYPE_EVENT = new(this, "Event", "Events", 1024, () => new MetaEvent());
+            META_TYPE_ACTION = new(this, "Action", "Actions", 512, () => new MetaAction());
+            META_TYPE_LANGUAGE = new(this, "Language", "Languages", 512, () => new MetaLanguage());
+            META_TYPE_TAG = new(this, "Tag", "Tags", 2048, () => new MetaTag());
+            META_TYPE_OBJECT = new(this, "ObjectType", "ObjectTypes", 512, () => new MetaObjectType());
+            META_TYPE_PROPERTY = new(this, "Property", "Properties", 512, () => new MetaProperty());
+            META_TYPE_GUIDEPAGE = new(this, "GuidePage", null, 512, () => new MetaGuidePage());
+            META_TYPE_DATA = new(this, "Data", null, 0, () => new MetaDataValue());
```

Comment:
Internal Data being treated as its own type is an issue, it's intentionally excluded from most systems